examples/elf/tests/errno/Makefile

```diff
@@ -86,6 +86,6 @@ clean:
 	$(call DELFILE, $(BIN))
 	$(call CLEAN)
 
-install:
+install: $(BIN)
```

Comment:
if we do this at this level, we should remove "build" step from examples/elf/tests/Makefile.

an alternative: https://github.com/apache/incubator-nuttx-apps/pull/103

Reply:
You are right. I think your approach is simple and more general.